### ISSUE TITLE
Specifying style can be a bit wordy

### ISSUE THREAD

**Issue by sakuraehikaru** [steamclock/niceComponents#49]
For example:
`DetailText(equity, style: Config.current.detailTextStyle.with(color: .platinum))`

It would be nice to find a way to write something like:
`DetailText(equity, style: .color(.platinum))`

Where the default behaviour is to use `Config.current` plus the corresponding style, and we only specify what's different; in this case just the color.

**Comment by brendanlensink** [comment]
This was created before 2.0, going to just close
